make/hotspot/lib/CompileGtest.gmk

```diff
@@ -110,7 +109,7 @@ $(eval $(call SetupJdkLibrary, BUILD_GTEST_LIBJVM, \
         self-assign-overloaded, \
     DISABLED_WARNINGS_clang_test_g1ServiceThread.cpp := delete-abstract-non-virtual-dtor, \
     DISABLED_WARNINGS_clang_test_logDecorations.cpp := missing-field-initializers, \
-    DISABLED_WARNINGS_microsoft := $(DISABLED_WARNINGS_microsoft), \
+    DISABLED_WARNINGS_microsoft := $(DISABLED_WARNINGS_microsoft) 4530, \
```

Comment:
How widespread are these warnings? I expect such a warning from jfr/test_networkUtilization.cpp,
but not anywhere else. Could the disable of this warning be narrowed accordingly?

Reply:
I checked. It is for all `test_*.cpp` files, due to gtest include files. So we need to to this broad spectrum disable.